style.css

```diff
@@ -0,0 +1,691 @@
+*,
+*::before,
+*::after {
+  box-sizing: inherit;
+  margin: 0;
+  padding: 0;
+}
+
+html {
+  box-sizing: border-box;
+  font-size: 62.5%;
+}
+
+body {
+  font-family: "Roboto Condensed", sans-serif;
+  background-color: whitesmoke;
+}
+
+.main {
+  background-color: #fff;
+}
+
+.navbars-container {
+  position: relative;
+}
+
+.navbar-top {
+  background-color: #f72210;
+  font-size: 1.5rem;
+  z-index: 11;
+  width: 100%;
+}
+
+.newsweek-logo {
+  background: #9b8583;
+}
+
+.btn-login {
+  background-color: #f72210;
+  font-weight: 700;
+  font-size: 1.5rem;
+}
+
+.btn-subscribe {
+  font-weight: 700;
+  font-size: 1.5rem;
+}
+
+.navbar-scrolling {
+  z-index: 10;
+  top: 0;
+  left: 0;
+  position: fixed;
+  width: 100%;
+}
+
+.logo-small {
+  height: 3rem;
+}
+
+.menu-hamburger {
+  width: 2.5rem;
+  height: 2.5rem;
+  cursor: pointer;
+  fill: whitesmoke;
+}
+
+.top-ul {
+  padding-bottom: 10px;
+  position: relative;
+  width: 100%;
+}
+
+.top-li {
+  display: inline-block;
+  list-style: none;
+  border-right: 1px solid gray;
+  padding: 0 1.5rem;
+  font-size: 2rem;
+  font-weight: 900;
+}
+
+.a-tag-li {
+  color: black;
+}
+
+.a-tag-li:hover {
+  color: red;
+  text-decoration: none;
+}
+
+.form {
+  width: 15rem;
+}
+
+.input-form {
+  border: none;
+  font-family: inherit;
+  font-size: 2rem;
+  margin-left: 1.5rem;
+  width: 10rem;
+  position: absolute;
+  transition: all 0.4s;
+}
+
+.input-form::placeholder {
+  font-size: 1.8rem;
+}
+
+.magnifying-glass {
+  transform: translate(9rem, 0.3rem);
+  width: 2.5rem;
+  height: 2.5rem;
+}
+
+.input-form:focus {
+  outline: none;
+  width: 120rem;
+  top: 0;
+  left: -1rem;
+}
+
+/* --------------------------NEWS NAVBAR--------------------------------- */
+
+.news-navbar-box {
+  padding: 0.2rem 0;
+  border-top: 1px solid grey;
+  border-bottom: 1px solid grey;
+  width: 100%;
+}
+
+.navbar-news {
+  color: #9b8583;
+  font-size: 1.5rem;
+}
+
+.news-nav-a-tag:hover {
+  color: #f72210 !important;
```

Comment:
Please, don't use !important.

Reply:
Hello, thanks for the feedback, we were able to improve our project thanks to it, best regards!